jetson/percep/pcl.cpp

```diff
@@ -404,19 +308,23 @@ bool PCL::CheckPath(std::vector<std::vector<int>> interest_points,
     #if PERCEPTION_DEBUG
```

Comment:
This is a lot of code in the PERCEPTION_DEBUG tag especially given it's just debug code. I'd suggest making a function to project the path in the "viewer".